vignettes/handling_incidence_objects.Rmd

```diff
@@ -17,88 +17,121 @@ knitr::opts_chunk$set(
 )
 ```
 
-`incidence()` objects are easy to work with, and we providing helper functions for both manipulating and accessing the underlying data and attributes.
-As `incidence()` objects are subclasses of [tibbles](https://tibble.tidyverse.org/) they also have good integration with [tidyverse](https://www.tidyverse.org/) verbs.
+`incidence()` objects are easy to work with, and we providing helper functions
+for both manipulating and accessing the underlying data and attributes.
 
 ## Modifying incidence objects
 
 ### `regroup()`
-Sometimes you may find you've created a grouped incidence but now want to change the internal grouping.
-Assuming you are after a subset of the grouping already generated, then you can use to `regroup()` function to get the desired aggregation:
+Sometimes you may find you've created a grouped incidence but now want to change
+the internal grouping. Assuming you are after a subset of the grouping already
+generated, then you can use to `regroup()` function to get the desired
+aggregation:
 
 ```{r regroup, message=FALSE}
 library(outbreaks)
-library(dplyr)
 library(incidence2)
+library(grates)
 
 # load data
 dat <- ebola_sim_clean$linelist
 
+# convert date_of_onset to isoweeks
+dat$date_of_onset <- as_isoweek(dat$date_of_onset)
+
 # generate the incidence object with 3 groups
-inci <- incidence(dat, date_of_onset, groups = c(gender, hospital, outcome), interval = "week")
-inci
+(inci <- incidence(dat, "date_of_onset", groups = c("gender", "hospital", "outcome")))
 
 # regroup to just two groups
-inci %>% regroup(c(gender, outcome))
+regroup(inci, c("gender", "outcome"))
 
 # drop all groups
-inci %>% regroup()
+regroup(inci)
 ```
 
 ### cumulate
 We also provide a helper function, `cumulate`() to easily generate cumulative 
 incidences:
 
 ```{r cumulate}
-inci %>% 
-  regroup(hospital) %>% 
-  cumulate() %>% 
-  facet_plot(n_breaks = 4, nrow = 3)
+x <- regroup(inci, "hospital")
+x <- cumulate(x)
+plot(x, nrow = 3)
 ```
 
 ### `keep_first()` and `keep_last()`
-Once your data is grouped by date, you may want to select the first or last few entries based on a particular date grouping using `keep_first()` and `keep_last()`:
+Once your data is grouped by date, you may want to select the first or last few
+entries based on a particular date grouping using `keep_first()` and
```

Comment:
'entries' is ambiguous, as it could mean the last non-zero data points; should it read the last 'dates'?

Reply:
This is a little tricky and also relates to #94. See discussion there.